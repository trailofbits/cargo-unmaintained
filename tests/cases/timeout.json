[
  {
    "name": "timeout",
    "outdated_deps": [],
    "repo_status": "Nonexistent",
    "version": "0.1.0"
  }
]
