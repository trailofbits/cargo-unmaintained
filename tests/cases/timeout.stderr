Scanning 1 packages and their dependencies (pass --verbose for more information)
